doc/command_ref.rst

```diff
@@ -1129,6 +1129,9 @@ packages to those matching the specification. All packages are considered if no
 ``--available``
     Limit the resulting set to available packages only (set by default).
 
+``--disable-modular-filtering``
+    Disables filtering of modular packages, so that packages of modules that aren't enabled are included in the result.
```

Comment:
@j-mracek can you please confirm I've used the correct formulation here? As in are those really packages of modules that aren't enabled? I'm not entirely sure.

Reply:
It can be either enabled stream or a default one if no other is enabled. In documentation, we use the term "active" (see https://dnf.readthedocs.io/en/latest/modularity.html#hotfix-repositories), but I haven't found any definition and as a user I wouldn't know what that means, so some note in parentheses or in the modularity documentation would be good.